lib/pf/task/fleetdm.pm

```diff
@@ -0,0 +1,393 @@
+package pf::task::fleetdm;
+
+=head1 NAME
+
+pf::task::fleetdm - Task for handling fleetdm events
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::task::fleetdm
+
+=cut
+
+use strict;
+use warnings;
+use pf::config qw(%Config);
+use base 'pf::task';
+use JSON;
+use HTTP::Tiny;
```

Comment:
Looks to be a new lib, most of the time curl is used in the code.

Reply:
I've notice we are using LWP::UserAgent in many times, 
also several occurance of "WWW::Curl::Easy" 

since HTTP::Tiny is also built-in ones, shall we switch to `LWP` or `WWW::Curl::Easy` ?